bin/cmds/up.js

```diff
@@ -23,6 +26,22 @@ exports.builder = (yargs) => {
         fs.accessSync(argv.f, fs.R_OK);
         return true;
       } catch (e) {
+        //if the config file is not found in cwd use comsiconfig to locate deployer.yml
+        //NOTE: in this fallback, the -f flag (if specified), for a custom config file, is ignored and uses the
+        //default "deployer.yml"
+        const result = explorerSync.search();
```

Comment:
I'd suggest that if the 'f' flag is set and no file is found, then don't fall back to using explorerSync but instead fall through to `throw new Error(err(`Argument check failed: ${argv.f} is not a readable file.`));`